### PR TITLE
Twin Blades and Gorehowl

### PR DESCRIPTION
**Pull request by obeest**
## Changelog:
- Added Illidan's Twin Blades of Azzinoth and gave them to him
- Added Gorehowl and gave it to Grom Hellscream

### CODE REVIEW

File: localisation/00_wc_artifact_localisation.csv [hunk 1]
```diff
@@ -65,6 +65,10 @@ twilight_canticle;Twilight Canticle;;Loblied des Zwielichts;;;;;;;;;;;x
...
+twin_blades_of_azzinoth;Twin Blades of Azzinoth;;Kriegsgleven von Azzinoth;;;;;;;;;;;
+twin_blades_of_azzinoth_desc;These twin-blades formerly belonged to Azzinoth, the leader of the doomguard, who was slain by Illidan 10.000 years ago. They are usually connected, but can be seperated and used as two weapons.;;Diese Doppelklingen geh�rten einst Azzinoth, dem Anf�hrer der Schreckenswache, der vor 10.000 Jahren von Illidan get�tet wurde. Normalerweise sind sie verbunden, k�nnen aber getrennt und als zwei Waffen benutzt werden.;;;;;;;;;;;
+gorehowl;Gorehowl;;Blutschrei;;;;;;;;;;;
+gorehowl_desc;Massive axe wielded by Grommash Hellscream - and his son Garrosh - of the Warsong Clan. It is permanently stained with the blood of the warrior's fallen enemies. Small holes and notches that dot the blade make it �sing� when swung.;;Eine riesige von Grommash Hellscream � und seinem Sohn Garrosh � des Warsong-Klans benutzte Axt. Sie ist permanent mit dem Blut gefallener Opfer des Kriegers beschmiert. Kleine L�cher und Einkerbungen an der Klinge lassen die Axt �singen� wenn sie geschwungen wird.;;;;;;;;;;;
```
Comment (MemeWizard) [inline localisation/00_wc_artifact_localisation.csv]: Are you sure localization should talk about events that didn't occur yet?

File: common/artifacts/901_wc_artifacts_weapon_twohanded.txt [hunk 2]
```diff
@@ -360,6 +360,72 @@ bloodhoof_runespear = {
...
+	indestructible = yes
+
+	martial = 1
+	intrigue = 1
+	combat_rating = 20
+
+	monthly_character_prestige = 1
+	
+	kaldorei_religion_opinion = -10
+	fel_group_opinion = 5
```
Comment (MemeWizard) [inline common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: So, for example, paladins can wield it and gain bonus opinion with the Legion?

Comment (geffe1) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: > So, for example, paladins can wield it and gain bonus opinion with the Legion?

Restrict them to warrior/rogue only perhaps?

Comment (arithon) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: or decide what class illidan will get, especially with TFT startdate. Demonhunter as class/subclass might be necessary

Comment (geffe1) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: > or decide what class illidan will get, especially with TFT startdate. Demonhunter as class/subclass might be necessary

I'd still let warrior/rogue use it due to the fact those two could use them in tbc if you somehow manage to kill Illidan and steal his stuff.

Comment (MemeWizard) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: > Restrict them to warrior/rogue only perhaps?

So, scarlet light warrior will get bonus opinion with the Legion?

Comment (geffe1) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: > > Restrict them to warrior/rogue only perhaps?
> 
> So, scarlet light warrior will get bonus opinion with the Legion?

That wasn't my point, but to which classes can use them.

Comment (zumbak04) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: Gosh, just restict the blades to `religion_group = fel_group`.

File: interface/wc_treasury.gfx [hunk 3]
```diff
@@ -487,4 +487,19 @@ spriteTypes = {
 		name = "GFX_bloodhoof_runespear_small"
 		texturefile = "gfx\\interface\\inventory\\artifacts\\bloodhoof_runespear_small.dds"
 	}
+	spriteType = {
+		name = "GFX_twin_blades_of_azzinoth"
+		texturefile = "gfx\\interface\\inventory\\artifacts\\twin_blades_of_azzinoth.dds"
+	}
+	spriteType = {
+		name = "GFX_twin_blades_of_azzinoth_small"
+		texturefile = "gfx\\interface\\inventory\\artifacts\\twin_blades_of_azzinoth_small.dds"
```
Comment (zumbak04) [inline interface/wc_treasury.gfx]: This .dds image doesn't exist. So, the small icon doesn't work.
<details>
<summary>Click to expand</summary>

image

</details>

File: interface/wc_treasury.gfx [hunk 4]
```diff
@@ -487,4 +487,19 @@ spriteTypes = {
...
+	}
+	spriteType = {
+		name = "GFX_gorehowl"
+		texturefile = "gfx\\interface\\inventory\\artifacts\\gorehowl.dds"
```
Comment (zumbak04) [inline interface/wc_treasury.gfx]: This .dds image doesn't exist. So, the small icon doesn't work.
<details>
<summary>Click to expand</summary>

image

</details>

File: interface/wc_treasury.gfx (hunk 3 above)
Comment (zumbak04) [inline interface/wc_treasury.gfx]: `twin_blades_of_azzinoth_small.dds` doesn't exist.

File: interface/wc_treasury.gfx [hunk 5]
```diff
@@ -487,4 +487,19 @@ spriteTypes = {
...
+	}
+	spriteType = {
+		name = "GFX_gorehowl_small"
+		texturefile = "gfx\\interface\\inventory\\artifacts\\gorehowl_small.dds"
```
Comment (zumbak04) [inline interface/wc_treasury.gfx]: `gorehowl_small.dds` doesn't exist.

**Comment by zumbak04** [comment]
For some reason, I can't open `gorehowl.dds` and `twin_blades_of_azzinoth.dds`. Have you used right editors to add them?
https://ck2.paradoxwikis.com/Graphics_modding#Editors
<details>
<summary>Click to expand</summary>

image

</details>

File: common/artifacts/901_wc_artifacts_weapon_twohanded.txt (hunk 2 above)
Comment (obeest) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: Done.

File: localisation/00_wc_artifact_localisation.csv [hunk 6]
```diff
@@ -65,6 +65,10 @@ twilight_canticle;Twilight Canticle;;Loblied des Zwielichts;;;;;;;;;;;x
...
+twin_blades_of_azzinoth;Twin Blades of Azzinoth;;Kriegsgleven von Azzinoth;;;;;;;;;;;
+twin_blades_of_azzinoth_desc;These twin-blades formerly belonged to Azzinoth, the leader of the doomguard, who was slain by Illidan 10.000 years ago. They are usually connected, but can be seperated and used as two weapons.;;Diese Doppelklingen geh�rten einst Azzinoth, dem Anf�hrer der Schreckenswache, der vor 10.000 Jahren von Illidan get�tet wurde. Normalerweise sind sie verbunden, k�nnen aber getrennt und als zwei Waffen benutzt werden.;;;;;;;;;;;
+gorehowl;Gorehowl;;Blutschrei;;;;;;;;;;;
+gorehowl_desc;Massive axe wielded by Grommash Hellscream of the Warsong Clan. It is permanently stained with the blood of the warrior's fallen enemies. Small holes and notches that dot the blade make it �sing� when swung.;;Eine riesige von Grommash Hellscream des Warsong-Klans benutzte Axt. Sie ist permanent mit dem Blut gefallener Opfer des Kriegers beschmiert. Kleine L�cher und Einkerbungen an der Klinge lassen die Axt �singen� wenn sie geschwungen wird.;;;;;;;;;;;
```
Comment (zumbak04) [inline localisation/00_wc_artifact_localisation.csv]:
```suggestion
gorehowl_desc;Massive axe wielded by Grommash Hellscream of the Warsong Clan. It is permanently stained with the blood of the warrior's fallen enemies. Small holes and notches that dot the blade make it "sing" when swung.;;Eine riesige von Grommash Hellscream des Warsong-Klans benutzte Axt. Sie ist permanent mit dem Blut gefallener Opfer des Kriegers beschmiert. Kleine L�cher und Einkerbungen an der Klinge lassen die Axt �singen� wenn sie geschwungen wird.;;;;;;;;;;;
```
IIRC, English not often use these quotation marks.
<details>
<summary>Click to expand</summary>

image

</details>

Comment (MemeWizard) [reply localisation/00_wc_artifact_localisation.csv]: 🤔 

image

Comment (arithon) [reply localisation/00_wc_artifact_localisation.csv]: @MemeWizard this error with quotation marks will be resolved in the next localisation crusade, already committed there.

**Pull request merged**